lab/lab2/js/part1-ajax-calls.js

```diff
@@ -103,12 +109,27 @@ var phillyBikeCrashesDataUrl = "https://raw.githubusercontent.com/CPLN692-MUSA61
   Remember to call all code within the function body. Use console.log to make sure
   that this step is completed before moving on!
 ===================== */
-
+var parsedData = JSON.parse(testing);
```

Comment:
you have defined this function `parseData` inside another function (the ajax call)

So when you try to call it on line 112, it can't find it.

![image](https://user-images.githubusercontent.com/8103418/53310762-2dfa7680-387c-11e9-96eb-f120853fdba3.png)